src/home/ExpandedCard.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
```

Comment:
I think Emily mentioned this in our last standup but is importing all of React necessary?